internal/eventstore/internal/repository/sql/data.go

```diff
@@ -0,0 +1,25 @@
+package sql
+
+import "database/sql/driver"
+
+// Data represents a byte array that may be null.
+// Data implements the sql.Scanner interface
+type Data []byte
```

Comment:
can't a byte array be null by default?

Reply:
the cockroach client unmarshals the given byte-slice and the go default unmarshaller returns an error if the byte is no valid json (with at least `{}`)